create-publication-from-scopus/build.gradle

```diff
@@ -0,0 +1,48 @@
+def problemVersion = '0.24.0'
+
+project.ext {
```

Comment:
Don't do this, rather add the generated code to the source set with e.g.

``` 
    project.ext {
        jaxbTargetDir = file("build/generated/sources/xjc")
    }
    sourceSets.main.java.srcDirs += jaxbTargetDir
```

Reply:
Hvorfor det? Vil det virker da å bruke klassene i unmarshalling?

Reply:
When generated code is in the build folder:
1) It is clear that it is generated code
2) The code is not versioned in git and therefore the PRs are much much smaller.
3) When you auto-generate the code from a remote XSD file then anytime our logic is not in accordance with the latest XSD file we will get a compile error instead of a runtime error. Compile errors are much easier. to fix.
4) Having generated code injected in your source code, might lead to leakages (i.e. more code outside this module will start using the generated code) This will lead to major problems when the XSD will be updated.

Reply:
Right. that is convincing.